content/en/blog/call-for-blogs.md

```diff
@@ -0,0 +1,42 @@
+# Call for blog contributions
+
+This is a constantly updated list of the blogs that the sig-docs is interested in.
```

Comment:
We should first set the context by stating which blog we're referring to. There are two, and only https://kubernetes.io/blog/ is within the remit of SIG Docs. It is one of our subprojects with its own Slack channel and set of reviewers/approvers.

Reply:
Could you provide some resources that I should go with to learn about those two types? I assumed there's only one type, Hence I referred it here.

Reply:
I mentioned it above, @NitishKumar06.
There is a blog on the Kubernetes Documentation website - https://kubernetes.io/blog.
There is another blog on the Kubernetes Contributor Community Website - https://www.kubernetes.dev/blog/

The first is the one within our remit.

Reply:
```suggestion
This is a list of the blog topics that the Kubernetes project is interested in.
```

- We don't “constantly update” any of our blog articles.
- If we did make a commitment to keep a page updated, I think it'd be something within https://k8s.dev/
- or possibly within https://kubernetes.io/docs/contribute/